api/filter/accesslog.proto

```diff
@@ -2,31 +2,58 @@ syntax = "proto3";
 
 package envoy.api.v2.filter;
 
+import "api/address.proto";
 import "api/base.proto";
 
 import "google/protobuf/duration.proto";
 import "google/protobuf/struct.proto";
 import "google/protobuf/timestamp.proto";
 import "google/protobuf/wrappers.proto";
 
-message AccessLogEntry {
-  // The HTTP request method (RFC 7231/2616)
-  RequestMethod request_method = 1;
+import "validate/validate.proto";
+
+
+// Envoy access logs describe incoming interaction with Envoy over a fixed
+// period of time, and typically cover a single request/response exchange,
+// (e.g. HTTP), stream (e.g. over HTTP/gRPC), or proxied connection (e.g. TCP).
+// Except where explicitly declared otherwise, all fields describe
+// *downstream* interaction between Envoy and a connected client.
+// Fields describing *upstream* interaction will explicitly include `upstream`
+// in their name.
+message AccessLogCommon {
+  // This field indicates the rate at which this log entry was sampled.
+  // Valid range is (0.0, 1.0].
+  double sample_rate = 1 [(validate.rules).double.gt = 0.0, (validate.rules).double.lte = 1.0];
+
+  // This field is the IP and port on which the request from the user was
+  // received.
+  SocketAddress destination_host = 2;
+
+  // Properties of the TLS connection, if used
```

Comment:
Please clean up all comments and make them "doc ready." (E.g., full sentences, periods end of sentences, etc.). Optimally, build the docs locally and do a full doc cleanup pass on this file. :)